bip32/src/lib.rs

```diff
@@ -8,6 +8,7 @@
     rust_2018_idioms,
     unused_qualifications
 )]
+#![feature(error_in_core)]
```

Comment:
This is a `stable` crate, as are all of the crates in this repo. We can’t merge nightly-only features.

Reply:
Oh, sorry. Let's wait until it's stable.